src/widget.rs

```diff
@@ -7,6 +7,15 @@ use serde_json::value::Value;
 
 use crate::themes::Theme;
 
+#[derive(Debug, Copy, Clone, Deserialize)]
+pub enum Spacing {
+    Normal,
+    /// Hide the leading space when the widget is inline
+    Inline,
+    /// Hide all spaces when the widget is hidden
```

Comment:
Just to make it clearer: "Hide both leading and trailing spaces when widget is hidden"